apis/core/lib/domain/organization/query.ts

```diff
@@ -58,3 +58,16 @@ export function cubeNamespaceAllowed(cube: NamedNode, organization: Term, client
     )
   }`.execute(client.query)
 }
+
+export function fetchOrganizationDatasets(organization: NamedNode, client = parsingClient): Promise<Quad[]> {
+  return CONSTRUCT`?s ?p ?o`
+    .WHERE`
+      GRAPH ?project {
+        ?project ${schema.maintainer} ${organization} .
+        ?project ${cc.dataset} ?dataset .
+      }
+      GRAPH ?dataset {
+        ?s ?p ?o .
+      }
+    `.execute(client.query)
```

Comment:
Just realized we will need to filter-in only datasets that are "published to opendata.swiss" and not expired.